_locales/_untranslated_en.json

```diff
@@ -44,5 +44,8 @@
   },
   "multidevice_connection_takes_too_long": {
     "message": "This takes longer than expected 🐌 make sure both devices are on the same Wi-Fi or network and check troubleshooting"
+  },
+  "one_result":  {
```

Comment:
the key says nothing about search or search term

Reply:
changed the wording according to the existing keys